README.md

```diff
@@ -191,27 +192,37 @@ There are two ways to specify the role: use the built-in role or create a custom
       - Microsoft.Compute/snapshots/delete
       - Microsoft.Compute/disks/beginGetAccess/action
       - Microsoft.Compute/disks/endGetAccess/action
-   
+
    Use the following commands to create a custom role which has the minimum required permissions:
-   ```
+
+   ```bash
    AZURE_ROLE=Velero
    az role definition create --role-definition '{
       "Name": "'$AZURE_ROLE'",
       "Description": "Velero related permissions to perform backups, restores and deletions",
       "Actions": [
+          "Microsoft.Storage/storageAccounts/blobServices/containers/read",
+          "Microsoft.Storage/storageAccounts/blobServices/generateUserDelegationKey/action",
           "Microsoft.Compute/disks/read",
           "Microsoft.Compute/disks/write",
           "Microsoft.Compute/disks/endGetAccess/action",
           "Microsoft.Compute/disks/beginGetAccess/action",
           "Microsoft.Compute/snapshots/read",
           "Microsoft.Compute/snapshots/write",
-          "Microsoft.Compute/snapshots/delete",
-          "Microsoft.Storage/storageAccounts/listkeys/action",
-          "Microsoft.Storage/storageAccounts/regeneratekey/action"
+          "Microsoft.Compute/snapshots/delete"
       ],
+      "notActions": [],
+      "dataActions": [
```

Comment:
After reverting some changes in https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure/pull/173, we don't need the actions listed in the `dataAction` part, so seems we don't need to make this change anymore.

Reply:
ok, I see... I just followed the instructions on `master` branch and works in Azure.